packages/core/src/Cardano/util/computeImplicitCoin.ts

```diff
@@ -179,7 +194,9 @@ export const computeImplicitCoin = (
     proposalProcedures
   );
 
-  const withdrawalsTotal = (withdrawals && BigIntMath.sum(withdrawals.map(({ quantity }) => quantity))) || 0n;
+  const hasRewardAccount = !!rewardAccounts?.length;
+  const withdrawalsAll = BigIntMath.sum(withdrawals.map(({ quantity }) => quantity));
+  const withdrawalsTotal = hasRewardAccount ? getOwnWithdrawals(withdrawals, rewardAccounts) : withdrawalsAll;
```

Comment:
Shouldn't this be 0? 🤔 
```suggestion
  const withdrawalsTotal = hasRewardAccount ? getOwnWithdrawals(withdrawals, rewardAccounts) : 0n;
```

Reply:
Initially we always filtered own withdrawals, but this is also used in txBuilder and due to some constraints, this logic should be returning all withdrawals like it does now. @AngelCastilloB could elaborate probably.

Reply:
Done